scripts/migrate/cli.ts

```
@@ -172,8 +182,12 @@ export function run(argv: readonly string[], log: (line: string) => void): Migra
   };
 }
 
+// Comparing against a basename split on '/' fails on Windows, where argv[1]
+// uses backslashes and the split yields the whole path -- the CLI then silently
+// declines to run. Normalising both sides to a file URL is platform-agnostic.
+const entrypoint = process.argv[1];
 const invokedDirectly =
-  process.argv[1] !== undefined && import.meta.url.endsWith(process.argv[1].split('/').pop() ?? '');
+  entrypoint !== undefined && import.meta.url === pathToFileURL(resolve(entrypoint)).href;
```

ESLint

[error] undefined is not allowed. Use null or proper type checking instead.

(no-restricted-syntax)